lib/brainstem/query_strategies/filter_and_search.rb

```diff
@@ -28,23 +38,28 @@ def run_search(scope, includes)
 
         result_ids, _ = @options[:primary_presenter].run_search(@options[:params][:search], search_options)
         if result_ids
-          result_ids
+          resulting_scope = scope.where(id: result_ids)
+          [resulting_scope, result_ids]
         else
           raise(SearchUnavailableError, 'Search is currently unavailable')
         end
       end
 
-      def paginate(scope)
-        if @options[:params][:limit].present? && @options[:params][:offset].present?
-          limit = calculate_limit
-          offset = calculate_offset
-        else
-          limit = calculate_per_page
-          offset = limit * (calculate_page - 1)
-        end
+      def ordering?
+        sort_name = @options[:params][:order].to_s.split(":").first
```

Comment:
Double check that you always have `[:order]` off `[:params]`

Reply:
```
Ruby:(2.3.3) object:(...FilterAndSearch>) >> @options[:params]
=> {"owned_by"=>"1", :search=>"toot, otto, toot"}
Ruby:(2.3.3) object:(...FilterAndSearch>) >> @options[:params][:order]
=> nil
Ruby:(2.3.3) object:(...FilterAndSearch>) >> @options[:params][:order].to_s
=> ""
Ruby:(2.3.3) object:(...FilterAndSearch>) >> @options[:params][:order].to_s.split(":")
=> []
Ruby:(2.3.3) object:(...FilterAndSearch>) >> @options[:params][:order].to_s.split(":").first
=> nil
```

I think we're good :+1:

Reply:
Cool